lib/pf/Portal/ProfileFactory.pm

```diff
@@ -33,6 +35,8 @@ Create a new pf::Portal::Profile instance based on parameters given.
 
 =cut
 
+tie our $PROFILE_FILTER_ENGINE , 'pfconfig::cached_scalar' => 'resource::ProfileFilterEngine';
```

Comment:
As we're starting from scratch for the filter engines namespaces, I think we should